mltrace/client.py

```diff
@@ -443,13 +433,27 @@ def get_recent_run_ids(limit: int = 5, last_run_id=None):
     return store.get_recent_run_ids(limit, last_run_id)
 
 
+def get_all_run_ids():
```

Comment:
This function doesn't need to exist, also `store.py` doesn't have the method `.get_all_run_ids`

Reply:
ah right.